static/css/starter-template.css

```diff
@@ -29,6 +29,9 @@ video {
 .hide-snippets {
   display: none !important;
 }
+.hide-options {
+  display:none !important;
+}
```

Comment:
This could be simplified to have a single "hide" class, and then add it or remove it to the specific sections.

Reply:
Looking at the code, we would have to add unique IDs to the sections, instead of unique classes.
I think using unique IDs is better semantically, but both methods work, so I leave that up to you.

Reply:
updated in https://github.com/bbcmicrobit/PythonEditor/pull/199/commits/2489a5a6d8ade073255dcca9dfeb9e06a9210d86 this unhides nav, button and sections for the existing flags and unhides anything with the class 'experimental' for anything else.